ui/pages/confirmations/send/send-content/add-recipient/domain-input.container.js

```diff
@@ -17,7 +17,7 @@ function mapDispatchToProps(dispatch) {
   return {
     lookupDomainName: debounce(
       (domainName) => dispatch(lookupDomainName(domainName)),
-      150,
+      450,
```

Comment:
Why did we increase the debounce here? And why 450?

Reply:
The snaps based resolution is a little slower, and I think the same value is used in another part of the code.
I'm planning a followup PR that should clean-up the state machine and I'll add constants for these.

Reply:
Ok, I get a bit of vertigo with magic numbers like these. Is it possible to find ourselves upon a race condition in this scenario?

Reply:
not really. there's a piece of code that resets the state if another lookup has already been requested.
The state machine is still a bit cumbersome and will be simplified a lot in the cleanup PR.
Most of the error states are no longer reachable, and some of the existing states are already irrelevant.